src/_colors.scss

```diff
@@ -48,6 +51,7 @@ $ui-light-primary: hsl(215, 100%, 95%);
 $ui-light-primary-transparent: hsla(215, 100%, 95%, 0);
 
 $ui-border: hsla(0, 0%, 85%, 1); //#D9D9D9
+$ui-border-light-blue: hsla(215, 50%, 90%, 1); //##D9E3F2
```

Comment:
A nitpick or two that you don't have to address:
1) the color here has a typo with two ##
2) the colors added above don't have a comment with the RGB hex value